### ISSUE TITLE
action type: mark-for-op should support days: 0

### ISSUE THREAD

**Issue by FireballDWF** [capitalone/cloud-custodian#1740]
Ask: change days exclusiveMinimum to False to enable setting days to zero.

The following action fails to validate
```
    actions:
      - type: mark-for-op
        op: delete        
        tag: custodian_status
        days: 0
```
with error:
```
Exception: Failed to validate on policy mark-rds-tagcompliance
 0 is less than or equal to the minimum of 0

Failed validating u'minimum' in schema[15][u'properties']['days']:
    {u'exclusiveMinimum': True, u'minimum': 0, u'type': u'integer'}

On instance['days']:
    0
```

This will enable policies which notify owners and mark-for-op in same policy, while allowing time, say 30-60 minutes, for Owner to tag instances before deletion.  We desire to delete quickly before people have spent time and labor with the resource, while still wanting to give them some time to tag, so and hourly unmark can occur before an hourly delete...

**Comment by kapilt** [comment]
sounds good

**Comment by scotwk** [comment]
Is this as simple as removing `exclusiveMinimum` from here?

https://github.com/capitalone/cloud-custodian/blob/master/c7n/tags.py#L537

**Comment by FireballDWF** [comment]
I expect the answer to be set it to False rather than removing it is the
right answer, but I'm not authoritative on this.

**Comment by scotwk** [comment]
@FireballDWF - it defaults to False, so removing it would be the same.

**Comment by joshuaroot** [comment]
Issue solved through #1815
